cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/CgmesExportContext.java

```diff
@@ -435,6 +450,24 @@ private static void addIidmMappingsEquivalentInjection(Network network) {
         }
     }
 
+    private void addIidmMappingsControlArea(Network network) {
+        CgmesControlAreas cgmesControlAreas = network.getExtension(CgmesControlAreas.class);
+        if (cgmesControlAreas == null) {
+            network.newExtension(CgmesControlAreasAdder.class).add();
+            cgmesControlAreas = network.getExtension(CgmesControlAreas.class);
+            String cgmesControlAreaId = CgmesExportUtil.getUniqueId();
+            cgmesControlAreas.newCgmesControlArea()
+                    .setId(cgmesControlAreaId)
+                    .setName("Network")
+                    .setEnergyIdentificationCodeEic("Network--1")
+                    .add();
+            CgmesControlArea cgmesControlArea = cgmesControlAreas.getCgmesControlArea(cgmesControlAreaId);
+            for (DanglingLine danglingLine : network.getDanglingLines()) {
+                cgmesControlArea.add(danglingLine.getTerminal());
+            }
+        }
+    }
+
```

Comment:
Could you explain why you're adding Control areas here? I don't think it is needed...

Reply:
The CGMES export was creating a "default" control area if none was defined in the input files, or the Network came from IIDM.
Instead of only writing it in the export module, we also keep it in the CGMES-IIDM mappings, as we are doing for the rest of information produced. The export module will write what is found in the CGMES-IIDM mapping data.

Reply:
Ok, thank you for your answer